vms/platformvm/txs/executor/staker_tx_verification.go

```diff
@@ -372,16 +357,8 @@ func verifyAddDelegatorTx(
 		return outs, nil
 	}
 
-	currentTimestamp := chainState.GetTimestamp()
-	// Ensure the proposed validator starts after the current timestamp
-	validatorStartTime := tx.StartTime()
```

Comment:
nit: can we restore the declarations here? we typically declare variables close to where they're used

Reply:
it was an attempt at reducing diffs but you're right, bette declare close to use